packages/ffe-core/less/colors.less

```diff
@@ -24,6 +24,7 @@
 // Purple
 @ffe-purple: #C94096;
 @ffe-purple-magenta: #A20076;
+@ffe-purple-visited: #551A8B;
```

Comment:
Navnet følger ikke konvensjonen vi har på alle andre farger, dvs. `-visited` beskriver bruk men ikke fargen.

Reply:
Fulgte eksemplet @ffe-blue-focus

Reply:
Kom gjerne med forslag til navn på denne lilla-varianten, som etter all sannsynlighet kun kommer til å brukes for :visited på lenker.

Reply:
Trenger den et navn hvis den bare skal brukes ett sted da? Alt som eksponeres som variabler i `colors.less` må regnes som fair game for andre å bruke.

Reply:
Trenger ikke ha et eget navn for min del. Kan alternativt hardkode fargen.